### DISCUSSION TITLE
Committers Activity Survey

### DISCUSSION THREAD

**Discussion by g1eny0ung** [chaos-mesh/chaos-mesh#4041] [General]
@chaos-mesh/maintainers @chaos-mesh/committers An important notice! 📣

Firstly, following https://github.com/chaos-mesh/chaos-mesh/pull/4039, we have successfully incorporated our prow bot named `chaotic`! Thanks to @STRRL for dedicating his free time and effort towards this achievement, it is truly appreciated.

This matter introduces a long-standing problem, **as most of our committers have been inactive for a long time.** When a PR is opened, the bot will automatically add several reviewers. Most of the time, these reviewers include people in our committers team. But due to inactive of them, **many of the PRs were not responded to promptly.**

I knew our team of committers was on the verge of death. 🙁 But according to the review process, **a PR should be seen by someone familiar with the changes, or passed on to someone familiar with it.**

Now, I'm trying to save the process. **This means that all committers should indicate whether they are willing to continue contributing in the future (for example, at least once a month, claim a PR or solve an existing problem, or participate in biweekly development meetings).**

So, I'd like to use 2 weeks to collect the survey, and if you want to stay involved in the development, please comment below, or I'll work with @STRRL to remove you from the committers team.

> But please note that this does not mean you will be removed from the team forever, this is all just to make our review process healthier, and if you ever come back, you will be included again. 🙂

I will additionally @ the following people since you are rarely active anymore for at least a half year (according to my memory, if I am wrong, please correct me in time 🙇🏼):

@Colstuwjx @Yiyiyimu @ethercflow @shivanshs9 @iguoyr @fewdan @Hexilee

And finally, FYI @STRRL @dcalvin

**Comment by STRRL** [comment]
As we have an "Emeritus Maintainers" section in https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md, we could also make a list of  "Emeritus Committers" for appreciating their contributions. ❤️

**Reply by g1eny0ung** [reply]
Makes sense. I agree with this approach.

**Comment by g1eny0ung** [comment]
This survey is over.

At the last dev meeting (2023-05-30), We decided to create the "Emeritus Committers" section and move all of the inactive committers into it. @cwen0 will help with this.

Again, thank you for your past contributions. ❤️

**Reply by cwen0** [reply]
> This survey is over.
> 
> At the last dev meeting (2023-05-30), We decided to create the "Emeritus Committers" section and move all of the inactive committers into it. @cwen0 will help with this.
> 
> Again, thank you for your past contributions. ❤️

I will create a pr to update these messages.

**Reply by g1eny0ung** [reply]
Ref: https://github.com/chaos-mesh/chaos-mesh/pull/4072.
